tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -220,7 +220,47 @@ async def test_wedge_command(self, bft_network):
         await self.verify_replicas_are_in_wedged_checkpoint(bft_network, checkpoint_before, range(bft_network.config.n))
         await self.verify_last_executed_seq_num(bft_network, checkpoint_before)
         await self.validate_stop_on_super_stable_checkpoint(bft_network, skvbc)
-    
+
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
+    async def test_unwedge_command(self, bft_network):
```

Comment:
The reason why this test is working (despite my comment from above) is because the operator sends a read command (which is not blocked on wedge).
While this is a valid solution, we want the unwedge command to be a write command due to 2 reasons:
1. We want to be able to record it on the blockchain (see https://github.com/vmware/concord-bft/blob/master/kvbc/include/reconfiguration_add_block_handler.hpp)
2. We want all replicas to get it together, and unlike write requests, read requests do not have this guarantee.